### PR TITLE
Translation in Japanese

### PR DESCRIPTION
**Pull request by zukizukizuki**
Japanese translation was performed.

## Things did
- Copy `en` folder
- Translated values in Json from English to Japanese

## Notes
English that can be understood in Japan is left as it is.

## Issue
https://github.com/privacy-scaling-explorations/website-v2/issues/122

### CODE REVIEW

**Comment by zukizukizuki** [comment]
@ChialiT 
please check it.

**Comment by zukizukizuki** [comment]
@ChialiT 
How's it going?

**Comment by ChialiT** [comment]
> @ChialiT How's it going?

Thank you for pinging me, I asked one of our member to review the translation. my Japanese is very limited.

**Comment by tkmct** [comment]
Hi @zukizukizuki, thank you for your contribution. I'm currently reviewing your translation. Bear with me for a moment please.

File: app/i18n/locales/jp/programs-page.json
Comment (tkmct) [inline app/i18n/locales/jp/programs-page.json:5]:
```suggestion
    "faq": "よくある質問",
```

File: app/i18n/locales/jp/programs-page.json
Comment (tkmct) [inline app/i18n/locales/jp/programs-page.json:6]:
```suggestion
    "curriculum": "カリキュラム",
```

File: app/i18n/locales/jp/programs-page.json
Comment (tkmct) [inline app/i18n/locales/jp/programs-page.json:7]:
```suggestion
    "howToApply": "応募方法",
```

File: app/i18n/locales/jp/programs-page.json
Comment (tkmct) [inline app/i18n/locales/jp/programs-page.json:8]:
```suggestion
    "applyNow": "今すぐ応募する",
```

File: app/i18n/locales/jp/programs-page.json
Comment (tkmct) [inline app/i18n/locales/jp/programs-page.json:9]:
```suggestion
    "learnMoreOnGithub": "GitHubで詳細を見る",
```

File: app/i18n/locales/jp/programs-page.json
Comment (tkmct) [inline app/i18n/locales/jp/programs-page.json:10]:
```suggestion
    "week": "第{{week}}週"
```

File: app/i18n/locales/jp/programs-page.json
Comment (tkmct) [inline app/i18n/locales/jp/programs-page.json:14]:
```suggestion
      "title": "必須条件",
```

File: app/i18n/locales/jp/programs-page.json
Comment (tkmct) [inline app/i18n/locales/jp/programs-page.json:16]:
```suggestion
        "コースの概要とリソース",
```

File: app/i18n/locales/jp/programs-page.json
Comment (tkmct) [inline app/i18n/locales/jp/programs-page.json:17]:
```suggestion
        "Git、GitHub、およびPRワークフローの基本",
```

File: app/i18n/locales/jp/programs-page.json
Comment (tkmct) [inline app/i18n/locales/jp/programs-page.json:18]:
```suggestion
        "ゼロ知識証明 (ZKPs) と数論の入門"
```

File: app/i18n/locales/jp/programs-page.json
Comment (tkmct) [inline app/i18n/locales/jp/programs-page.json:22]:
```suggestion
      "title": "暗号学の基礎",
```

File: app/i18n/locales/jp/programs-page.json
Comment (tkmct) [inline app/i18n/locales/jp/programs-page.json:24]:
```suggestion
        "Circom入門",
```

File: app/i18n/locales/jp/programs-page.json
Comment (tkmct) [inline app/i18n/locales/jp/programs-page.json:25]:
```suggestion
        "暗号化とハッシュ関数の基礎",
```

File: app/i18n/locales/jp/programs-page.json
Comment (tkmct) [inline app/i18n/locales/jp/programs-page.json:26]:
```suggestion
        "デジタル署名と楕円曲線暗号"
```

File: app/i18n/locales/jp/programs-page.json
Comment (tkmct) [inline app/i18n/locales/jp/programs-page.json:30]:
```suggestion
      "title": "より高度な暗号技術とZKP",
```

File: app/i18n/locales/jp/programs-page.json
Comment (tkmct) [inline app/i18n/locales/jp/programs-page.json:32]:
```suggestion
        "Circom 集中講座 + 実践",
```

File: app/i18n/locales/jp/programs-page.json
Comment (tkmct) [inline app/i18n/locales/jp/programs-page.json:33]:
```suggestion
        "KZGコミットメントと zkSNARKs",
```

File: app/i18n/locales/jp/programs-page.json
Comment (tkmct) [inline app/i18n/locales/jp/programs-page.json:34]:
```suggestion
        "トラステッドセットアップと Groth16 の概要"
```

File: app/i18n/locales/jp/common.json
Comment (tkmct) [inline app/i18n/locales/jp/common.json:3]:
```suggestion
  "siteDescription": "暗号学研究とコレクティブな実験を通じてイーサリアムをよりよくする",
```

**Review by tkmct — CHANGES_REQUESTED** [review]
Thank you very much for your contribution. I added suggestions in the PR.
Please take a look and accept those suggestions. If you need any help, let us know.

**Comment by zukizukizuki** [comment]
@tkmct 
thank you for your review!
All imported.

File: app/i18n/locales/jp/about-page.json
Comment (tkmct) [inline app/i18n/locales/jp/about-page.json:3]:
```suggestion
  "description": "PSEはイーサリアムファウンデーションが支援する研究開発ラボで、暗号プリミティブのユースケースを試すことに専念しています。私たちは、オープンソースのインフラやツールの開発、ワークショップやサマープログラムの開催、そして暗号学を現実のものにしようとする人々とのコラボレーションなどを行います。",
```

**Review by tkmct — CHANGES_REQUESTED** [review]
I'm sorry, I missed one change. Can you apply the suggested change too?

**Review by tkmct — APPROVED** [review]
Thanks! LGTM.

**Comment by zukizukizuki** [comment]
@tkmct (@ChialiT) 
thank you!Waiting for approval.

**Comment by AtHeartEngineer** [comment]
@tkmct Do you think this is ready to merge?

**Review by ChialiT — APPROVED** [review]

**Comment by ChialiT** [comment]
@AtHeartEngineer can u approve this one so i can merge it?thanks

**Review by AtHeartEngineer — APPROVED** [review]

**Pull request merged**
